java/src/com/google/idea/blaze/java/fastbuild/FastBuildServiceImpl.java

```diff
@@ -353,7 +354,8 @@ private FastBuildState.BuildOutput buildDeployJar(
                   .filter(artifact -> artifact.getArtifactPath().endsWith(deployJarLabel.targetName().toString()))
                   .collect(Collectors.toUnmodifiableSet()),
               BlazeContext.create(),
-              project);
+              project,
+              RuntimeArtifactKind.JAR);
```

Comment:
Let's not merge this yet. I want to check this first, since these configurations are currently also broken due to an AOSP regression.